### ISSUE TITLE
Issues with accessibility in vaadin application containing grid

### ISSUE THREAD

**Issue by gauravbh1** [vaadin/framework#9008]
- **Vaadin version** :7.6.8
- **Description of the bug**
   I am using a Vaadin grid in my application. I use the server side api for the same. 
   I am running into some accessibility violations while testing the app with IBM's Dynamic Assement  
   plugin  extension on chrome
 [https://chrome.google.com/webstore/detail/dynamic-assessment-plugin/aahpafpbmmgednbflpalchnlbicaeppi]
  I use the 508 ruleset in the above extension.
  
  
Here are the violations that I am running into
1) Frames and IFrames must have a non-empty title attribute.
2) Each form control must have associated label. 
3) For a data table (i.e., the table tag does not contain a WAI-ARIA 'presentation' role or 'none' role), identify headers for the table rows and columns.
4) Combine color and descriptive markup to convey information.
5) Frames that contain content that is not HTML may not be accessible.
6) Do not use ::before and ::after pseudo-elements to insert non-decorative content.

**- If possible, minimal reproducible example**
   When I test a vaadin demo page (http://demo.vaadin.com/sampler/#ui/grids-and-trees/grid/multi-select) using the above extension, I run into similar accessibility violations. 

Can these violations be handled in anyway or is this a genuine bug or limitation of Vaadin?

Thanks,
Gaurav

**Comment by TatuLund** [comment]
Duplicate of https://github.com/vaadin/framework/issues/7913
